app/scripts/views/mixins/email-first-experiment-mixin.js

```diff
@@ -13,7 +13,7 @@ define(function (require, exports, module) {
   'use strict';
 
   const ExperimentMixin = require('views/mixins/experiment-mixin');
-  const EXPERIMENT_NAME = require('lib/experiments/grouping-rules/email-first').name;
+  const EXPERIMENT_NAME = 'emailFirst';
```

Comment:
Since the static name getter is removed to fix Safari, I just hard coded the name here.